src/portable/st/stm32_fsdev/dcd_stm32_fsdev.c

```diff
@@ -258,12 +258,8 @@ void dcd_int_enable (uint8_t rhport)
 {
   (void)rhport;
 #if defined(STM32F0)
-  NVIC_SetPriority(USB_IRQn, 0);
   NVIC_EnableIRQ(USB_IRQn);
```

Comment:
Should we be enabling/disabling only the USB IRQ? Or all of them? Do we have a rule that application code's interrupts cannot call USB-related functions? I think that the RTOS will use global interrupt disabling to avoid issue, so it only applies to the NO OS option.

Reply:
tinyUSB doesn't and couldn't impose that rule, those tinyUSB will only use dcd_int_enable/disable by itself. Usually for dcd_init() and  accessing tud task event queue (mutex). DCD should only enable/disable those only needed to get it worked.